.github/workflows/dbt_slim_ci.yml

```
@@ -56,7 +56,7 @@ jobs:
         run: "dbt seed --select state:modified --state ."
 
       - name: dbt run initial model(s)
-        run: "dbt -x run --select state:modified --state ."
+        run: "dbt -x run --select state:modified --state . --full-refresh"
```

I suppose this can be removed here? I will make the suggested edit, and we can remove it once we see a passing CI.
```suggestion
        run: "dbt -x run --select state:modified --state ."
```